tools/runtime-stats-monitor.md

```diff
@@ -0,0 +1,215 @@
+---
+id: RNSM
+title: Runtime Network Stats Monitor
+sidebar_label: Runtime Network Stats Monitor
+---
+
+The **Runtime Network Stats Monitor (RNSM)** is a configurable component to display various network stats in an on-screen display at runtime, including stats from the Netcode for GameObjects (Netcode) package and custom, user-defined stats.
+
+## Prerequisites
+
+To enable the Runtime Network Stats Monitor, [install the Multiplayer Tools package](install-tools.md). 
+
+You should now see the Runtime Network Stats Monitor from the **Unity Hub** top menu bar under **Component** > **Netcode**, or when searching for it by name after pressing **Component** > **Add..**. 
+
+:::note
+You must have an object selected for these options to be available.
+:::
+
+## Adding the Runtime Network Stats Monitor to a Scene
+
+To add the RNSM to your scene, you must add it as a component to an object:
+
+1. Select an object in your scene. <- SHOULD AN OBJECT BE MADE SPECIFICALLY FOR THE RNSM? LIKE BEST PRACTICE-WISE? EMPTY OBJECT OR A SPECIFIC TYPE
+2. Add the RNSM commponent by either:
+   1. Selecting **Add Component** in the **Inspector** tab, navigate to **Netcode** and select **RuntimeNetStatsMonitor**.
+   OR
+   2. From the top menu bar, go to **Component** > **Netcode** > **RuntimeNetStatsMonitor**.
+
+## Configuring
+
+You can configure the RNSM to your liking with its [Display Elements](#display-elements), [Panel Settings](#panel-settings), and [Styling](#styling-the-runtime-network-stats-monitor). Each of these settings can be found from the **Inspector** tab of the RNSM component. The RNSM package comes with default settings for each option that can be used "as is" or as guidance for creating your own.
+
+:::note
+The default setting for the Display Elements, Panel Settings, and Styling are greyed out and not adjustable by default. To make modifications or build your own, you will need to create your own configuration presets. Instructions for each customization are below.
+:::
+
+### Display Elements
+
+Each Display Element displays one or more stats that can be added, removed, or modified under the element's **Stats** field.
+
+To use the default Display Elements:
+
+1. It may already be the default in the **Configuration** field of the **Inspector** tab of the RNSM component.
+
+OR
+
+1. From the **Project** tab, go to **Packages** > **Multiplayer Tools** > **NetStatsMonitor** > **Assets** > **Configurations**. The default configuration for the RNSM is in this folder.
+2. Select the `GameObject` with the RNSM component so that it appears in the **Inspector** tab.
+3. Drag the default configuration asset from the **Project** tab to the **Configuration** field under the **RuntimeNetStatsMonitor** component of your `GameObject`.
+
+To configure your own Display Elements presets:
```

Comment:
Nit, I think it might help draw attention to this section if it had a 4th level heading or bold text for **"Using a Custom Display Elements Configuration"**